client/app/project/project.controller.js

```diff
@@ -403,9 +404,9 @@
                 addAoiToMap(vm.projectData.areaOfInterest);
                 addPriorityAreasToMap(vm.projectData.priorityAreas);
                 addProjectTasksToMap(vm.projectData.tasks, true);
+                loadAnnotations(id);
```

Comment:
I know you're still working on the front end side, but figured I would comment this while it's fresh.

Loading on a project with stored annotations works fine and is hardly noticeable in that process. However, if a project does _not_ have annotations, the 404 thrown doesn't seem to be caught well by the frontend and it thinks there is no project data, causing e.g. the project text to revert to `undefined`--until the next refresh of project data (without refresh of  annotations) after 10 seconds.

Reply:
Done